arbor/backends/gpu/threshold_watcher.hpp

```diff
@@ -110,7 +110,8 @@ class threshold_watcher {
     /// crossed since current time t, and the last time the test was
     /// performed.
     void test(array* time_since_spike) {
-        arb_assert(values_);
+        // either number of cvs is 0 or values_ is not null
+        arb_assert((n_cv_ == 0) || (bool)values_);
```

Comment:
it does unfortunately occur in at least one of the tests: `simulation.null`.

Reply:
Ok, then we should guard against this in `fvm_layout` and either (silently??) drop these cells there or
throw `empty_cell_error` since this can't possibly intended.

Reply:
I am not going to fix this here, creating a small PR is better imo.

Reply:
Agreed.